src/main/resources/processes/FOI_ACCEPTANCE.bpmn

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
On the out mapping here, can't we just have AcceptanceTeam instead of ${execution.getVariable("AcceptanceTeam")} ?

Reply:
I've made this change and it still works as expected.